src/Api/Polls.elm

```diff
@@ -38,6 +45,10 @@ type alias Poll =
     }
 
 
+type alias PollIdentifier =
```

Comment:
`PollDiscriminator` is maybe a better name, if we want to have indicate it's the discriminating field of the weak entity.